boards/arm/pan1780_evb/pan1780_evb.dts

```diff
@@ -0,0 +1,258 @@
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		evb_sw1: evb_sw_1 {
+			gpios = <&gpio0 11 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW1 on EVB";
+		};
+		evb_sw2: evb_sw_2 {
+			gpios = <&gpio0 12 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW2 on EVB";
+		};
+		evb_sw3: evb_sw_3 {
+			gpios = <&gpio0 24 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW3 on EVB";
+		};
+		evb_sw4: evb_sw_4 {
+			gpios = <&gpio0 25 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW4 on EVB";
+		};
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 3 0>, /* A0 */
+					<1 0 &gpio0 4 0>, /* A1 */
+					<2 0 &gpio0 28 0>, /* A2 */
+					<3 0 &gpio0 29 0>, /* A3 */
+					<4 0 &gpio0 30 0>, /* A4 */
+					<5 0 &gpio0 31 0>, /* A5 */
+					<6 0 &gpio1 1 0>, /* D0 */
+					<7 0 &gpio1 2 0>, /* D1 */
+					<8 0 &gpio1 3 0>, /* D2 */
+					<9 0 &gpio1 4 0>, /* D3 */
+					<10 0 &gpio1 5 0>, /* D4 */
+					<11 0 &gpio1 6 0>, /* D5 */
+					<12 0 &gpio1 7 0>, /* D6 */
+					<13 0 &gpio1 8 0>, /* D7 */
+					<14 0 &gpio1 10 0>, /* D8 */
+					<15 0 &gpio1 11 0>, /* D9 */
+					<16 0 &gpio0 12 0>, /* D10 */
+					<17 0 &gpio0 13 0>, /* D11 */
+					<18 0 &gpio0 14 0>, /* D12 */
+					<19 0 &gpio0 15 0>, /* D13 */
+					<20 0 &gpio0 26 0>, /* D14 */
+					<21 0 &gpio0 27 0>; /* D15 */
+	};
+
+	arduino_adc: analog-connector {
+		compatible = "arduino,uno-adc";
+		#io-channel-cells = <1>;
+		io-channel-map = <0 &adc 1>,	/* A0 = P0.3 = AIN1 */
+						 <1 &adc 2>,	/* A1 = P0.4 = AIN2 */
+						 <2 &adc 4>,	/* A2 = P0.28 = AIN4 */
+						 <3 &adc 5>,	/* A3 = P0.29 = AIN5 */
+						 <4 &adc 6>,	/* A4 = P0.30 = AIN6 */
+						 <5 &adc 7>;	/* A5 = P0.31 = AIN7 */
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &evb_led1;
+		led1 = &evb_led2;
+		led2 = &evb_led3;
+		led3 = &evb_led4;
+		sw0 = &evb_sw1;
+		sw1 = &evb_sw2;
+		sw2 = &evb_sw3;
+		sw3 = &evb_sw4;
+		bootloader-led0 = &evb_led1;
+	};
+};
+
+&adc {
+	status = "okay";
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uarte";
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-1 = <&uart0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+arduino_serial: &uart1 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart1_default>;
+	pinctrl-1 = <&uart1_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+arduino_i2c: &i2c0 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-1 = <&i2c0_sleep>;
+	pinctrl-names = "default", "sleep";
+};
+
+&i2c1 {
+	compatible = "nordic,nrf-twi";
+	/* Cannot be used together with spi1, so disabled by default. */
+	status = "disabled";
```

Comment:
nit: I think SoC dtsi files already flag this one as disabled, so a comment should be sufficient.

Reply:
Fixed